config/settings/settings.yml

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
die Datei kommt nicht ins git. Statt Dessen wäre eine `config/settings/settings.yml.template` gut in der Beispielhaft steht was die App automatisiert in die Datei reinschreibt.